.github/workflows/slow-tests.yml

```
@@ -0,0 +1,32 @@
+name: Slow Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - v[0-9]+\.[0-9]+-dev
```

actionlint (1.7.11)

[error] character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)